js/search/controllers/search.controller.js

```diff
@@ -56,6 +84,34 @@
                 autoStart: clickedOnPlay || platform.isMobile || typeof startTime !== 'undefined'
             });
         }
+
+        /**
+         * Are there any items left to show
+         * @returns {boolean}
+         */
+        function itemsLeft() {
+            return  !!(vm.feed && (searchFeed.playlist.length > vm.feed.playlist.length));
+        }
+
+        /**
+         * @ngdoc method
+         * @name jwShowcase.search.SearchController#showMoreClickHandler
+         * @methodOf jwShowcase.search.SearchController
+         *
+         * @description
+         * Handle click event on the show more button
+         */
+        function showMoreClickHandler () {
+            var feedPlaylistLength = vm.feed.playlist.length;
+            
```

Comment:
Remove newline

Reply:
The first line  (105) is a shorthand for vm.feed.playlist.length. 

The second line (107) is the result of a calculation. The two lines should be divided by a newline

Reply:
Both are declarations.

Reply:
Had a discussion. And agree.